exchanges/stream/websocket_connection.go

```diff
@@ -22,34 +22,32 @@ import (
 // SendMessageReturnResponse will send a WS message to the connection and wait
 // for response
 func (w *WebsocketConnection) SendMessageReturnResponse(signature, request interface{}) ([]byte, error) {
```

Comment:
I have an extra request. Could you update `TestSendMessageReturnResponse` to the following?
```go
func TestSendMessageReturnResponse(t *testing.T) {
	t.Parallel()
	wc := &WebsocketConnection{
		Verbose:          true,
		URL:              "wss://ws.kraken.com",
		ResponseMaxLimit: time.Second * 5,
		Match:            NewMatch(),
	}
	if wc.ProxyURL != "" && !useProxyTests {
		t.Skip("Proxy testing not enabled, skipping")
	}

	err := wc.Dial(&dialer, http.Header{})
	if err != nil {
		t.Fatal(err)
	}

	go readMessages(t, wc)

	request := testRequest{
		Event: "subscribe",
		Pairs: []string{currency.NewPairWithDelimiter("XBT", "USD", "/").String()},
		Subscription: testRequestData{
			Name: "ticker",
		},
		RequestID: wc.GenerateMessageID(false),
	}

	_, err = wc.SendMessageReturnResponse(request.RequestID, request)
	if err != nil {
		t.Error(err)
	}

	// with timeout
	wc.ResponseMaxLimit = 1
	_, err = wc.SendMessageReturnResponse("123", request)
	assert.ErrorIs(t, err, errMatchTimeout, "SendMessageReturnResponse should error when request ID not found")
}
```
Where errMatchTimeout is
```go
var errMatchTimeout = errors.New("websocket connection: timeout waiting for response with signature")
```

Reply:
Hello Shazbert, thank you for making those changes, however please rename the test as it is out-of-sync with the function name